types/jasminewd2/jasminewd2-tests.ts

```diff
@@ -143,7 +143,6 @@ describe('jasminewd', () => {
       };
       jasmine.addMatchers(matchers);
       jasmine.getEnv().addMatchers(matchers);
-      jasmine.getEnv().currentSpec.addMatchers(matchers);
```

Comment:
we should probably add `paths` resolution, so the tests passes with older version, instead of removing this test, what do you think?

Reply:
I gave that a quick try just now and it looks like the switch from triple-slash require to import will probably require a significant rewrite of the definition file similar to what happened to jasmine-node in this PR. I think it's less risky to remove that one line. It looks like jasminewd2 (not my package, btw) always required Jasmine 2, and `jasmine.getEnv().currentSpec` only worked in Jasmine 1.

Reply:
Agree,thx!